presto-benchto-benchmarks/src/main/resources/benchmarks/presto/tpch.yaml

```diff
@@ -0,0 +1,37 @@
+datasource: presto
+query-names: presto/tpch/${query}.sql
+runs: 6
```

Comment:
If we would like to have values for `runs, prewarmup-runs, database` and etc. customized, how we do it?